src/views/SearchResults/index.tsx

```diff
@@ -0,0 +1,65 @@
+import { Box, SimpleGrid, Flex, GridItem, Tag, Text } from "@chakra-ui/react";
+import { useEffect, useMemo } from "react";
+import { Link } from "react-router-dom";
+import { fetchPackages } from "../../api/package/packages";
+import { useQuery } from "../../hooks/useQuery";
+import { useRequest } from "../../hooks/useRequest";
+
+export function SearchResults() {
+  const query = useQuery().get("q") ?? "";
+  const [requestPackages, { data }] = useRequest(fetchPackages);
+
+  useEffect(() => {
+    void requestPackages();
+  }, [requestPackages]);
+
+  const results = useMemo(() => {
+    /* const searcher = new FuzzySearch(packagesResponse.data?.packages ?? [], [ */
+    /*   "name", */
+    /* ]); */
+
+    /* return query ? searcher.search(query) : []; */
+    return data?.packages.filter((item) =>
+      JSON.stringify(item).includes(query)
```

Comment:
I wonder if it makes sense to revise how we implement this search. I think at a basic level we could just filter by `name.toLowerCase().includes(query.toLowerCase())` which would be less expensive than JSON.stringifying every item. We could even expand on this by doing something similar to github's filters with other assembly properties such as `keywords` or `author`.

A few ways we could do this - 
a: github style: `keyword:eks author:foo @foo/eks-cdk-package`

b: Search bar only for names, add tags for key - value filters (don't have a good ex off the top of my head, though I've seen it in a few places before)

Reply:
I actually used the `fuzzy-search` library at first but ended up doing this as it's what `awscdk.io` currently uses. I agree this isn't a long term solution but for now I think this is mostly about setting up the lifecycle of input -> submit -> show results page.

Reply:
& one more thought here - if the query is an empty string we can just return `data?.packages ?? []` as a slight optimization

Reply:
so the current implementation is if the query is empty, show all results. I say we stick with that for now.

Reply:
I agree with that - I just mean that we can skip the filter + JSON.stringify on the package list since we know we'll just return everything!

Reply:
ahhh gotcha, yeah sounds good!